R/reprex_inform.R

```diff
@@ -0,0 +1,22 @@
+reprex_quiet <- function() {
+  as.logical(Sys.getenv("REPREX_QUIET", unset = "NA"))
+}
+
+local_reprex_quiet <- function(reprex_quiet, env = parent.frame()) {
+  withr::local_envvar(c(REPREX_QUIET = reprex_quiet), .local_envir = env)
+}
+
+reprex_inform <- function(message) {
+  quiet <- reprex_quiet() %|% is_testing()
```

Comment:
The inclusion of `is_testing()` is a concession to the truly staggering amount of messages that otherwise bubble up from the tests now.

Reply:
What's `%|%`?

Reply:
`%|%` is to `NA` as `%||%` is to `NULL`. Well, sort of. It's this operator in rlang:

https://rlang.r-lib.org/reference/op-na-default.html